File: app/javascript/channels/comments_channel.js
import consumer from "./consumer"

$(document).on('turbolinks:load', function() {
    consumer.subscriptions.create({ channel: "CommentsChannel", question_id: gon.question_id }, {
        received(data) {
            if (gon.user_id === data.user_id) return;  //здесь нехватает проверки на пользователя, чтобы дважды не добавлять коммент

            const template = require('./templates/comment.hbs')
            if (data['commentable_type'] == 'Question') {
                $('#question-'+ gon.question_id +' .comments').append(template(data))
            } else {
                $('.answers #answer-id-'+ data['commentable_id'] +' .comments').append(template(data))
            }
        }
    });
});

File: app/channels/comments_channel.rb
class CommentsChannel < ApplicationCable::Channel
  def subscribed
    stream_for "comments_#{params[:question_id]}"
  end

  def follow(data)
    unfollow
    stream_from "question_#{data['id']}_comments"
  end

  def unfollow
    stop_all_streams
  end
end
